Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 Language Features:
  * Introduce syntax for specifying contract storage layout base.
+ * Code Generator: generate code for contracts with a specified storage layout base.
```

Comment:
This is the same feature, just split into multiple PRs. There should be one changelog entry.

Reply:
Yeah, was wondering if that should be the case.
Fixed.